src/pages/aclmanagement.html

```diff
@@ -23,17 +23,23 @@ <h1 id="acl-management-view-title">
                 </button>
             </div>
             <ul class="nav nav-tabs acl-tabs">
-                <li class="resource-tab nav-item">
-                    <span class="nav-link" ng-click="switchTab(ACL_SCOPE.STATEMENT)" ng-class="activeTabScope === ACL_SCOPE.STATEMENT ? 'active':''"><span class="dirty-scope text-primary" ng-if="dirtyScope.has(ACL_SCOPE.STATEMENT)">*</span>{{'acl_management.rulestable.tab.statement' | translate}}</span>
+                <li class="statement-tab nav-item">
+                    <span class="nav-link"
+                          ng-click="switchTab($event, ACL_SCOPE.STATEMENT)"
+                          ng-class="{'active': activeTabScope === ACL_SCOPE.STATEMENT, 'disabled': editedRuleIndex !== undefined}"><span>{{'acl_management.rulestable.tab.statement' | translate}}</span><i class="text-primary icon-exclamation" ng-if="dirtyScope.has(ACL_SCOPE.STATEMENT)"></i></span>
```

Comment:
Have you noticed what his `disabled` class did with the tabs look?
![image](https://github.com/Ontotext-AD/graphdb-workbench/assets/51084653/11d80981-e1d6-4627-ab14-2d73205a9e85)

Reply:
There is also a row height which you set on the table rows which seems to be calculated probably wrong 
![image](https://github.com/Ontotext-AD/graphdb-workbench/assets/51084653/c76a34e8-b61d-466c-96f4-e648b0a1ca16)

Reply:
fixed the background color